tests/tests_ssh.yml

```diff
@@ -48,18 +61,33 @@
       delegate_to: "{{ kdump_test_ssh_server_outside }}"
       delegate_facts: true
 
-    - include_role:
-        name: linux-system-roles.kdump
-      vars:
-        kdump_ssh_user: >-
-          {{ hostvars[kdump_test_ssh_server_outside]['ansible_user_id'] }}
-        # This is the outside address. Ansible will connect to it to
-        # copy the ssh key.
-        kdump_ssh_server: "{{ kdump_test_ssh_server_outside }}"
-        kdump_path: /tmp/test
-        kdump_target:
-          type: ssh
-          # This is the ssh dump server address visible from inside
-          # the machine being configured. Dumps are to be copied
-          # there.
-          location: "{{ kdump_ssh_user }}@{{ kdump_test_ssh_server_inside }}"
+    - name: Print message that this test is skipped on EL 6
+      debug:
+        msg: Skipping the test on EL 6 because control node == managed node
+      when:
+        - ansible_distribution in ['CentOS','RedHat']
+        - ansible_distribution_major_version == '6'
+        - kdump_test_ssh_server_outside == inventory_hostname
```

Comment:
not sure I understand what is going on here - afaik we don't have any tests where the control node is EL6

Reply:
Right, I misunderstood this initially, it is not control node == managed node, it's ssh_server_outside == localhost. Changed this to `Skipping the test on EL 6 when storing logs over ssh to localhost`.

The issue here is that kdump on EL 6 is quite old, and `mkdumprd` in it does not expect storing logs over ssh on itself (localhost). Our tests are single-host, so when we configure storing logs over ssh to localhost, `mkdumprd` fails to find the correct interface.